tests/console/yast2_lan_hostname.pm

```diff
@@ -25,9 +25,9 @@ sub hostname_via_dhcp {
     $cmd{hostname_dns_tab} = 'alt-s';
     $cmd{home}             = 'home';
     $cmd{spc}              = 'spc';
-
-    y2logsstep::yast2_console_exec(yast2_module => 'lan');
-    accept_warning_network_manager_default;
+    y2_module_consoletest::yast2_console_exec(yast2_module => 'lan');
+    y2_module_consoletest::accept_warning_network_manager_default;
+    y2_module_basetest::accept_warning_network_manager_default;
```

Comment:
I guess you've mistakenly called the `accept_warning_network_manager_default` function twice. Could you please remove the duplication.

Reply:
Done